scene/2d/animated_sprite_2d.cpp

```diff
@@ -390,8 +396,9 @@ bool AnimatedSprite2D::is_playing() const {
 	return playing;
 }
 
-void AnimatedSprite2D::play(const StringName &p_animation, const bool p_backwards) {
-	backwards = p_backwards;
+void AnimatedSprite2D::play(const StringName &p_animation, bool p_backwards) {
+	play_reversed = p_backwards;
+	backwards = signbit(speed_scale) != play_reversed;
```

Comment:
It is nitpick, but it is better to match the name of the argument with the name of the variable to be held. Caching is fine for now.

```suggestion
	backwards = p_backwards;
	is_backwards_currently = signbit(speed_scale) != play_reversed;
```

Reply:
Renamed more appropriately.